core/plugins/org.polarsys.capella.core.data.core.properties/plugin.xml

```diff
@@ -97,6 +97,12 @@
                id="org.polarsys.capella.core.data.core.properties.sections.StringPropertyValueSection"
                tab="BaseCapella">
          </propertySection>
+         <propertySection

```

Comment:
There's no need to contribute to BaseCapella since the title block only appears on diagram. Same remark applies for BaseSiriusTable.

Reply:
Removed.